package.json

```diff
@@ -26,33 +26,29 @@
   "author": "GoogleCloudPlatform",
   "license": "Apache-2.0",
   "dependencies": {
-    "@actions/core": "^1.5.0",
-    "fs": "0.0.1-security",
-    "google-auth-library": "^7.9.2",
-    "uuidv4": "^6.2.12"
+    "@actions/core": "^1.6.0",
+    "google-auth-library": "^7.10.3",
+    "yaml": "^1.10.2"
```

Comment:
YAML was a dev dep, but it's actually used in the gkeClient

Reply:
huh, weird it didnt complain